miracl/reg/miracl_reg_clar-allen.sh

```diff
@@ -463,7 +463,7 @@ function resampleclar()
 	ifdsntexistrun ${resclar} "Resampling CLARITY input" \
 	ResampleImage 3 ${inclar} ${resclar} ${vox}x${vox}x${vox} ${ifspacing} ${interp}
 
-	c3d ${resclar} -type ushort -o ${resclar}
+	c3d ${resclar} -type ushort -o final_${resclar}
```

Comment:
Yep, I will do that in the future. The problem was that I was changing the code and Ahmadreza was testing it so I needed to push it for him to test it since we weren't developing in the same directory.